gce_vm_image_import.Dockerfile

```diff
@@ -12,8 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/distroless/base
+FROM debian:stretch
 
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -q -y curl qemu-utils gnupg
+RUN echo "deb http://packages.cloud.google.com/apt gcsfuse-stretch main" > /etc/apt/sources.list.d/gcsfuse.list
+COPY gcsfuse-apt-key.gpg .
```

Comment:
Why is this file needed?

Reply:
That's the GPG key for gcfuse's apt repo: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/installing.md